src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
apply shorthand

Reply:
`background: url(...), no-repeat, center;` -does not work. What syntax is correct?